lexicons/com/atproto/admin/defs.json

```diff
@@ -368,6 +380,145 @@
         "height": { "type": "integer" },
         "length": { "type": "integer" }
       }
+    },
+    "subjectReviewState": {
+      "type": "string",
+      "knownValues": ["#reviewOpen", "#reviewEscalated", "#reviewClosed"]
+    },
+    "reviewOpen": {
+      "type": "token",
+      "description": "Moderator review status of a subject: Open. Indicates that the subject needs to be reviewed by a moderator"
+    },
+    "reviewEscalated": {
+      "type": "token",
+      "description": "Moderator review status of a subject: Escalated. Indicates that the subject was escalated for review by a moderator"
+    },
+    "reviewClosed": {
+      "type": "token",
+      "description": "Moderator review status of a subject: Closed. Indicates that the subject was already reviewed and resolved by a moderator"
+    },
+    "modEventTakedown": {
+      "type": "object",
+      "description": "Take down a subject permanently or temporarily",
+      "properties": {
+        "durationInHours": {
+          "type": "integer",
+          "description": "Indicates how long the takedown should be in effect before automatically expiring."
+        },
+        "expiresAt": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Indicates at what time the subject's takedown will be/was reverted."
+        }
+      }
+    },
+    "modEventReverseTakedown": {
+      "type": "object",
+      "description": "Revert take down action on a subject",
+      "properties": {
+        "comment": {
+          "type": "string",
+          "description": "Describe reasoning behind the reversal."
+        }
+      }
+    },
+    "modEventComment": {
+      "type": "object",
+      "description": "Add a comment to a subject",
+      "required": ["comment"],
+      "properties": {
+        "comment": {
+          "type": "string"
+        },
+        "persistNote": {
+          "type": "boolean",
+          "description": "Make the comment a persistent note on the subject"
+        }
+      }
+    },
+    "modEventReport": {
+      "type": "object",
+      "description": "Report a subject",
+      "required": ["reportType"],
+      "properties": {
+        "comment": {
+          "type": "string"
+        },
+        "reportType": {
+          "type": "ref",
+          "ref": "com.atproto.moderation.defs#reasonType"
+        }
+      }
+    },
+    "modEventLabel": {
+      "type": "object",
+      "description": "Apply/Negate labels on a subject",
+      "required": ["createLabelVals", "negateLabelVals"],
+      "properties": {
+        "createLabelVals": {
+          "type": "array",
+          "items": { "type": "string" }
+        },
+        "negateLabelVals": {
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      }
+    },
+    "modEventFlag": {
```

Comment:
I'm all for removing it until we've nailed the use-case, personally!  I originally added it so that PDS operators would have an action that is a negative signal but has no effect so they could use to implement their own complex policies (e.g. a three-strikes policy).

Reply:
I'd be ok with mapping `#flag` to `#modEventAcknowledge`.